Gemfile

```diff
@@ -13,9 +13,9 @@ group :test do
   gem "chefstyle"
   gem "rake"
   gem "rspec", "~> 3.0"
-  if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.5")
-    gem "ohai", "<15"
-    gem "chef", "<15"
+  if Gem::Version.new(RUBY_VERSION) < Gem::Version.new("2.6")
+    gem "chef-zero", "~> 14.0"
+    gem "chef", "~> 15.0"
```

Comment:
hmmm, i haven't been doing that, but that's not a bad idea to get coverage on chef-15, i just ran a few of those by hand